spec/basic_parsing_spec.rb

```diff
@@ -177,7 +177,7 @@
       nte = HL7::Message::Segment::NTE.new
       msg << nte
       nte2 = HL7::Message::Segment::NTE.new
-      msg << nte
+      msg << nte2
```

Comment:
See, warnings **are** useful, there was actually a "bug" hiding here :stuck_out_tongue_winking_eye:  (`nte2` was assigned, but `nte` was used on the next line instead)